### ISSUE TITLE
HTTP/1.1 cannot be used for Consul with two discovers

### ISSUE THREAD

**Issue by stevehu** [networknt/light-4j#634]
In the open-banking demo application, the accounts will call the balances and transactions for more information. I have the following calls in the constructor of the handler. 

```
            transactionHost = cluster.serviceToUrl("https", "com.networknt.ob.transactions-3.1.2", tag, null);
            balanceHost = cluster.serviceToUrl("https", "com.networknt.ob.balances-3.1.2", tag, null);

```
And got the error documented in https://github.com/open-banking/light-config-test/issues/3

After switching to http2 to true, everything works.

**Comment by stevehu** [comment]
In reality, it doesn't make sense to connect to the Consul using anything other than HTTP/2 as both side supports it. However, the consul.yml default value is false and that might lead users to use it by default even they are not using an agent on the localhost. It might be just a documentation update to ensure the mistake won't be made.

**Comment by stevehu** [comment]
This is resolved along with #649
